File: src/main/resources/export/java/Command-requires.java
#set($command = $helper.getByName($command_name, $robot))
#if ($command.getProperty("Requires").getValue() != "None")

        m_#variable($command.getProperty("Requires").getValue()) = subsystem;
        addRequirements(m_#variable($command.getProperty("Requires").getValue()));
#else

        // m_subsystem = subsystem;
        // addRequirements(m_subsystem);    
#end

File: src/main/resources/export/cpp/Command-requires.cpp
#set($command = $helper.getByName($command_name, $robot))
#set($first = 1)
#if ($command.getProperty("Requires").getValue() != "None")
	#set($cmd1 = ${command.getProperty("Requires").getValue()})
	#set($cmd2 = ${cmd1.toLowerCase()})
	#set($cmd = "m_${cmd2}")
	#if($first)
## #variable(${cmd})(#variable(${cmd2}))
AddRequirements(#variable(${command.getProperty("Requires").getValue()}));
		#set($first = 0)
	#else
		## AddRequirements(#variable(${command.getProperty("Requires").getValue()}));
,#variable(${cmd})(#variable(${cmd2}))
	#end
#end
